streamer/parser/metadata.md

```diff
@@ -291,45 +293,66 @@ Does not apply.
 
 The integer is the value of the `meta` element whose `property` attribute has the value `schema:numberOfPages`.
 
-## Rendition
+## Rendition / Presentation
 
-The `rendition` of a publication is an object containing the following keys: 
+The `presentation` of a publication is an object containing the following keys: 
 
-- `overflow`;
+- `continuous`;
 - `layout`;
 - `orientation`;
+- `overflow`;
 - `spread`.
 
-The value of each key is a string that must conform to the allowed values in the EPUB specification.
+In addition, the following elements may be included in `properties` in a Link Object from the `readingOrder` (spine overrides):
+
+- `layout`;
+- `orientation`;
+- `overflow`;
+- `page`;
+- `spread`.
 
 ### Flow
 
-The `rendition:flow` metadata is mapped to `overflow`.
+The `rendition:flow` metadata is mapped to `overflow` and `continuous`.
 
 The `overflow` of a publication is a key whose value can be the following string: 
 
 - `auto`;
 - `paginated`;
-- `scrolled`;
-- `scrolled-continuous`.
+- `scrolled`.
+
+This `overflow` can be `continuous`, which is a key whose value is a boolean: `true` or `false`
 
 #### EPUB 2.x
 
 Does not apply.
 
 #### EPUB 3.x
 
+##### Global Property
+
 The string is the value of the `<meta>` element whose `property` attribute has the value `rendition:flow` with the following mapping:
 
-| rendition:flow     | value               |
-|--------------------|---------------------|
-| auto               | auto                |
-| paginated          | paginated           |
-| scrolled-doc       | scrolled            |
-| scrolled-continous | scrolled-continuous |
+| rendition:flow      | values                                        |
+|---------------------|-----------------------------------------------|
+| auto                | "overflow": "auto" + "continuous": false      |
+| paginated           | "overflow": "paginated" + "continuous": false |
+| scrolled-doc        | "overflow": "scrolled" + "continuous": false  |
+| scrolled-continuous | "overflow": "scrolled" + "continuous": true   |
```

Comment:
@HadrienGardeur re `continuous: true` default change, isn’t the mapping table already covering it?

Reply:
Yeah, seems so. I'll merge it now in this case.